src/common/FundFilter/FundFilter.js

```diff
@@ -1,25 +1,19 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import { SelectionFilter } from '@folio/stripes-acq-components';
 
-import {
-  getFundOptions,
-} from '../utils';
-import { fundsShape } from '../shapes';
-
 const FundFilter = ({ funds, ...rest }) => {
-  const options = getFundOptions(funds);
```

Comment:
I actually don't think you need to touch this file, or maybe you have a reason for that?

Reply:
Sorry, but I'd like to clarify. Which file do you mean? `getFundOptions.js` or `FundFilter.js`?

Reply:
both, please look at my another comment for clarification

Reply:
both files were rolled to initial state back